CSS/style.css

```diff
@@ -0,0 +1,399 @@
+:root {
+  --ff-cocogoose: 'COCOGOOSE', sans-serif;
+  --ff-lato: 'Lato', sans-serif;
+  --color-black: #272a31;
+  --color-red: #ec5242;
+  --color-lightgrey: #d3d3d3;
+  --color-darkgrey: #5e5e5e;
+  --color-whitesmoke: #f7f7f7;
+  --color-white: #fff;
+}
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: var(--ff-lato);
+}
+
+body {
+  background: var(--color-whitesmoke);
+}
+
+header {
+  height: 6.5rem;
+}
+
+.menu {
+  display: block;
+  top: 5%;
+  left: 10%;
+  transform: translate(-50%, -50%);
+  position: absolute;
+  width: 30px;
+}
+
+.menu::before,
+.menu::after,
+.menu div {
+  background: var(--color-darkgrey);
+  content: "";
+  display: block;
+  height: 4px;
+  border-radius: 1px;
+  margin: 7px 0;
+  transition: 0.5s;
+}
+
+.first {
+  display: none;
+  height: 30%;
+  justify-content: flex-end;
+  padding-right: 168px;
+  align-items: center;
+  background: var(--color-black);
+}
+
+li,
+ul {
+  display: inline;
+  list-style-type: none;
+  padding: 0 15px;
+  color: var(--color-white);
+  cursor: pointer;
+  text-align: center;
+}
+
+li:hover {
+  color: var(--color-red);
+}
+
+a {
+  text-decoration: none;
+}
+
+.fa {
+  color: var(--color-white);
+  position: absolute;
+  height: 10%;
+}
+
+.fa:hover {
+  top: 0;
+  color: var(--color-red);
+  transition: 0.5s;
+}
+
+select {
+  color: var(--color-white);
+  border: none;
+  background-color: transparent;
+  cursor: pointer;
+}
+
+select:hover {
+  color: var(--color-red);
+  transition: 0.5s;
+}
+
+.second {
+  display: none;
+  justify-content: space-around;
+  align-items: center;
+  height: 80%;
+  background: var(--color-white);
+}
+
+#Home {
+  color: var(--color-red);
+}
+
+.logo {
+  width: 100px;
+  height: 30px;
+  background: var(--color-white);
+  padding: 10px 20px;
+  cursor: pointer;
+}
+
+.second a {
+  color: var(--color-black);
+  text-align: center;
+}
+
+a:hover {
+  color: var(--color-red);
+}
+
+#mawazine {
+  color: var(--color-red);
+  border: 2px solid var(--color-red);
+  padding: 5px;
+  text-transform: uppercase;
+}
+
+main {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 1rem;
+  background-image: url(../assets/main_big.png);
```

Comment:
- [ ] [OPTIONAL[ - Great job in making the requested changes and adding the background image. It seems like the main reason why the background image is not showing is because of this wrong file path. double-check the folder name is "Assets" and not "assets". This should be the correct line: ```  background-image: url(../Assets/main_big.png);``` . Kindly note that different operating systems resolve file paths differently and your OS (presumable windows) is not strict on the file paths.
````